<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="openFileButton">
         <property name="text">
          <string>Открыть файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="placeRoundedButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Выстроить по кругу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restoreColorsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Востановить цвета</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Обход графа</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Задержка:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="delaySpinBox">
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.250000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="deepthTrRadio">
            <property name="text">
             <string>Обход в глубину</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="widthTrRadio">
            <property name="text">
             <string>Обход в ширину</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startTraversalButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Начать обход</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="outputField">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
